src/components/Layout/TopNav.tsx

```diff
@@ -169,7 +169,7 @@ function TopNav() {
       >
         <div className="mx-auto flex w-full max-w-[120rem] items-center justify-between p-6">
           <Link to="/webui/dashboard" className="flex items-center gap-x-3">
-            <ShokoIcon className="w-8" />
+            <ShokoIcon className="w-23" />
```

Comment:
Why are we increasing the width of the SVG by 3?

Reply:
done, now it seems slightly on the larger side though I am confused as to why it worked with w-23